megamek/src/megamek/common/Targetable.java

```diff
@@ -128,6 +128,11 @@ default boolean isBomber() {
      * @return
      */
     public boolean isEnemyOf(Entity other);
+    
+    public default boolean isHexBeingBombed() {
+        return getTargetType() == TYPE_HEX_AERO_BOMB || 
+                getTargetType() == TYPE_HEX_BOMB;
+    }
```

Comment:
Huh.  i didn't know Java 8 allowed us to do this now.  This used to be a compile-time error.  Also, `public` isn't needed.